### ISSUE TITLE
compiling error with the latest svn (1792)

### ISSUE THREAD

**Issue by aircrack-ng** [aircrack-ng/aircrack-ng#800]
**Reported by Jano on 31 Oct 2010 18:17 UTC**

```
besside-ng.c: In function do_wep_crack:
besside-ng.c:1895: error: ignoring return value of write, declared with attribute warn_unused_result
make[*** [besside-ng.o](1]:) Errore 1
make[uscita dalla directory /home/jano/aircrack-ng/src
make: *** [all](1]:) Errore 2
```

Bye Jano

**Comment by aircrack-ng** [comment]
**Comment by Jano on 31 Oct 2010 18:17 UTC**

The problem occurs after this change:
http://trac.aircrack-ng.org/changeset/1789

Jano

**Comment by aircrack-ng** [comment]
**Comment by misterx on 31 Oct 2010 18:17 UTC**

Fixed by aircrack-ng/aircrack-ng@b1eb828a62dead5b637304e09a50e560ff4bd06a.

**Comment by aircrack-ng** [comment]
**Modified by misterx on 31 Oct 2010 18:17 UTC**
